lib/macmillan/utils/rspec/matchers/be_valid_mosaic.rb

```diff
@@ -0,0 +1,43 @@
+require 'rspec/expectations'
+
+### Example:
+### it { is_expected.to be_valid_mosaic }
+#
+### example output:
+#
+### Failure/Error: it { is_expected.to be_valid_mosaic }
+### Missing required property: template in /layout
+
+RSpec::Matchers.define :be_valid_mosaic do
```

Comment:
Actually I forget that it is relatively easy to get it on Jenkins. 

@ettomatic is there a better place to put this then?